assignments/assignment0.md

```diff
@@ -5,14 +5,14 @@ nav_exclude: true
 ---
 
 # GIT Instructions
-
+### Please read these assignment instructions in their entirety very carefully before starting it!
 ## 1. Set Up Your GitHub Repository:
 - Create a new repository on GitHub named `username.github.io`, where `username` is your GitHub username.
 - This repository will host your personal website.
 
 ## 2. Design Your Website:
 - Create a simple HTML/CSS website. You can use any website template you like or build your own from scratch.
-- Ensure that your website includes the following sections:
+- Ensure that your website includes the following sections with `About Me as the landing page of your website`:
   - **About Me**: This will be your landing page. A brief introduction about yourself. The About me section should be accessible at `username.github.io/about-me`, where `username` is your GitHub username.
```

Comment:
```suggestion
  - **About Me**: This will be your landing page. A brief introduction about yourself. The About me section should be accessible at `username.github.io/`, where `username` is your GitHub username.
```